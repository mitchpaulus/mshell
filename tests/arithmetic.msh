2 3 * str wl
1.5 2.5 * str wl
5 2 / str wl
5.0 2.0 / str wl
5 2.0 / str wl
5.0 2 / str wl

-3 abs str wl

2 3 - str wl

1.5 0.5 - str wl
2 1.5 - str wl
2.5 1 - str wl
