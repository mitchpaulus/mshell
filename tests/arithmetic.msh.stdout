6
3.75
2
2.5
3
-1
1
0.5
1.5
